pages/common/rabbitmqctl-cluster.md

```diff
@@ -0,0 +1,32 @@
+# rabbitmqctl-cluster
+
+> Manage RabbitMQ nodes in a cluster.
+> More information: <https://www.rabbitmq.com/rabbitmqctl.8.html>.
+
+- Display the status of the cluster:
+
+`rabbitmqctl cluster_status`
+
+- Display the status of the current node:
+
+`rabbitmqctl status`
+
+- Start the RabbitMQ application on a specific node:
+
+`rabbitmqctl [--node {{nodename}}] start_app`
+
+- Stop the RabbitMQ application on a specific node:
+
+`rabbitmqctl [--node {{nodename}}] stop_app`
+
+- Stop a specific RabbitMQ node:
+
+`rabbitmqctl [--node {{nodename}}] stop`
+
+- Reset a specific RabbitMQ node to a clean state:
+
+`rabbitmqctl [--node {{nodename}}] reset`
```

Comment:
```suggestion
`rabbitmqctl --node {{nodename}} start_app`

- Stop the RabbitMQ application on a specific node:

`rabbitmqctl --node {{nodename}} stop_app`

- Stop a specific RabbitMQ node:

`rabbitmqctl --node {{nodename}} stop`

- Reset a specific RabbitMQ node to a clean state:

`rabbitmqctl --node {{nodename}} reset`
```

Reply:
We do not use optionality outside of placeholders. Either the user needs to use it or they don't. Be explicit with your examples so that the user get the exact command they need.